### PR TITLE
TEST: skip gtest team_size=1 w/o tl/self

### PR DESCRIPTION
**Pull request by vspetrov**
## What
skip gtest cases for team_size == 1 if tl/self is not available

## Why ?
"UCC_TLS=ucp ./gtest" crashes

### CODE REVIEW

File: test/gtest/common/test_ucc.cc [hunk 1]
```diff
@@ -501,6 +502,12 @@ const std::vector<UccTeam_h> &UccJob::getStaticTeams()
                                std::end(staticTeamSizes));
     if (0 == staticTeams.size()) {
         for (auto ts : teamSizes) {
+            if (ts == 1 && !tl_self_available()) {
+                /* don't use team_size = 1 if there is no tl/self.
+                   we can't modify nStaticTeams, so just use some other
+                   team_size */
+                ts = 3;
```
Comment (kingchc) [inline test/gtest/common/test_ucc.cc:509]: Can't we just skip if ts is 1? (i.e., do `continue` here)

Comment (vspetrov) [reply test/gtest/common/test_ucc.cc:509]: no, as comment says above, tests assume there nStaticTeams available, so we need to create all of them

**Review by kingchc — APPROVED** [review]
LGTM

**Review by manjugv — APPROVED** [review]

**Comment by vspetrov** [comment]
bot:retest

**Pull request merged**